cmd/snap-failure/cmd_snapd.go

```diff
@@ -153,9 +158,14 @@ func (c *cmdSnapd) Execute(args []string) error {
 	if err != nil && !os.IsNotExist(err) {
 		logger.Noticef("snapd socket still exists before restarting socket service, but unable to remove: %v", err)
 	}
-	output, err = exec.Command("systemctl", "restart", "snapd.socket").CombinedOutput()
-	if err != nil {
-		return osutil.OutputErr(output, err)
+
+	restartCmd := exec.Command("systemctl", "restart", "snapd.socket")
+	restartCmd.Env = os.Environ()
+	restartCmd.Stdout = Stdout
+	restartCmd.Stderr = Stderr
+
+	if err := restartCmd.Run(); err != nil {
+		logger.Noticef("failed to restart snapd.socket: %v", err)
 	}
 
```

Comment:
hmm I guess I'm not sure of the interaction between calling `reset-failed` on  `snapd.failure.service`, since that's the service we are running as. I would want to test this before making that change here